server/svix-server/migrations/20220428190104_message_expiration.up.sql

```diff
@@ -0,0 +1,5 @@
+BEGIN;
```

Comment:
I'm pretty sure it runs in a transaction anyway, you don't need to add one manually.